firebase-app-distribution/src/main/AndroidManifest.xml

```diff
@@ -44,12 +44,12 @@
 
         <provider
             android:name="androidx.core.content.FileProvider"
-            android:authorities="${applicationId}.provider"
+            android:authorities="${applicationId}.appdistro.fileprovider"
```

Comment:
Sorry for the late review, but I think we should change this to `firebaseappdistribution.fileprovider` because we don't use `appdistro` externally, and the firebase prefix helps customers quickly identify this authority as part of the Firebase ecosystem (and aligns with the name of the main folder for this SDK)